pipelines/controller/src/main/java/com/google/fhir/analytics/PipelineManager.java

```diff
@@ -601,13 +627,23 @@ public void run() {
           Thread.activeCount());
       String currentDwhRoot = null;
       try {
-        currentDwhRoot = options.getOutputParquetPath();
+        if (runMode != RunMode.VIEWS) {
+          currentDwhRoot = options.getOutputParquetPath();
+        } else {
+          currentDwhRoot = options.getParquetInputDwhRoot();
+        }
+
         List<PipelineResult> pipelineResults =
             EtlUtils.runMultiplePipelinesWithTimestamp(pipelines, options);
         // Remove the metrics of the previous pipeline and register the new metrics
         manager.removePipelineMetrics();
         pipelineResults.stream()
             .forEach(pipelineResult -> manager.publishPipelineMetrics(pipelineResult.metrics()));
+        if (runMode == RunMode.VIEWS) {
+          // Nothing more is needed to be done as we do not recreate a new DWH in this mode.
+          // TODO record timing info and other details in this case.
+          return;
```

Comment:
In case of error, currently the logs are captured in the current DWH root location and they are also viewable from the UI. This should be thought for the `recreate view` scenario.

Reply:
I see, that's a good point. So I actually checked some error scenarios and the only change that I made was to overwrite the `error.log` file if one already exists. We can make this smarter and create files in sequence, e.g., `error1.log`, `error2.log`, etc. but I think the current solution is good enough; WDYT?

Reply:
I think its good enough for now as the error file returns the latest error log. We can think about the multiple error logs scenario if there is a need for it.